nixos/modules/system/boot/systemd/tmpfiles.nix

```diff
@@ -254,5 +276,62 @@ in
       "R! /nix/var/nix/gcroots/tmp           -    -    -    - -"
       "R! /nix/var/nix/temproots             -    -    -    - -"
     ];
+
+    boot.initrd.systemd = {
+      additionalUpstreamUnits = [
```

Comment:
Something we've neglected to add so far has been `systemd-tmpfiles-setup-dev-early.service`, so now would be a good time to do that.

Reply:
We pull that in already, but in [boot/systemd.nix](https://github.com/NixOS/nixpkgs/blob/102b1152a6ea69f4be9d66bde3cb3141d5642b6c/nixos/modules/system/boot/systemd.nix#L56). I'll move it here to tmpfiles.nix, seems appropriate.